src/net/sourceforge/kolmafia/KoLmafia.java

```diff
@@ -130,13 +132,25 @@ public abstract class KoLmafia {
     System.setProperty("com.apple.mrj.application.live-resize", "true");
     System.setProperty("com.apple.mrj.application.growbox.intrudes", "false");
     System.setProperty("java.net.preferIPv4Stack", "true");
+    // override content types to avoid a Rhino problem
+    // see https://github.com/mozilla/rhino/issues/1232
+    ensureContentTypes();
 
     if (SwinglessUIUtils.isSwingAvailable()) {
       JEditorPane.registerEditorKitForContentType("text/html", RequestEditorKit.class.getName());
     }
     System.setProperty("apple.laf.useScreenMenuBar", "true");
   }
 
+  private static void ensureContentTypes() {
+    var contentTypesFile = KoLConstants.DATA_LOCATION.toPath().resolve("content-types.properties");
+    if (!Files.exists(contentTypesFile)) {
+      FileUtilities.loadLibrary(
+          KoLConstants.DATA_LOCATION, KoLConstants.DATA_DIRECTORY, "content-types.properties");
+    }
+    System.setProperty("content.types.user.table", contentTypesFile.toString());
+  }
+
```

Comment:
So, if the file is missing/removed, we just ignore it.

That seems reasonable.